__tests__/createSchema.spec.js

```diff
@@ -1,37 +1,91 @@
-import { createSchema, normalized } from '../src';
+import { createSchema, normalized, createIndex } from '../src';
 
 describe('createSchema specification', () => {
   it('checks for basic usage', () => {
     const schema = createSchema('scheme');
-    const reducer = schema.reducer([]);
-    let state = reducer(undefined, schema.insert({ id: 1, name: 'John Doe' }));
-    expect(normalized.all(state)).toEqual([{ id: 1, name: 'John Doe' }]);
-    state = reducer(state, schema.insert({ id: 2, name: 'Jane Doe' }));
-    state = reducer(state, schema.insert({ id: 3, name: 'J Doe' }));
+    const index = createIndex('departmentId', 'positionId');
+    const initalValue = {
+      id: 1,
+      name: 'John Doe',
+      departmentId: 3,
+      positionId: 2,
+    };
+    const reducer = schema.reducer([initalValue], [index]);
+
+
+    let state = reducer(undefined, schema.insert({
+      id: 2, name: 'Lore Epsum', departmentId: 3, positionId: 5,
+    }));
+
+    expect(normalized.all(state)).toEqual([
+      initalValue,
+      {
+        id: 2, name: 'Lore Epsum', departmentId: 3, positionId: 5,
+      },
+    ]);
+
```

Comment:
Add an index.allIds(state) value check, which is what we would be using most of the time.